common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -250,7 +252,51 @@ private static URL getResource(String path, ClassLoader loader) {
             case 1:
                 return urlsList.get(0);
             default:
-                throw new IllegalStateException("Multiple resources found for '" + path + "': " + urlsList);
+                try {
+                    MessageDigest md = MessageDigest.getInstance("SHA-256");
```

Comment:
Why not MD5?

Reply:
IMHO it doesn't really matter and md5 was flagged as insecure (which also not really matters tho)

Reply:
Alright, that makes sense. :)